### ISSUE TITLE
ini vs properties

### ISSUE THREAD

**Issue by utoddl** [cmur2/joe-syntax#26]
Could you comment a bit about the differences between ini.jsf and properties.jsf? From what I've read, "properties files" are mostly a rebranding of "ini files". I'm considering some changes to ini.jsf, particularly in how it deals with errors. (it seems to handle correct .ini files just fine.) Before I start working on it, I'd like to make sure I understand the contexts of these two .jsf files, as they seem to cover largely the same problem space. Thanks.

**Comment by cmur2** [comment]
Yes, these syntaxes are mostly the same given that they deal with key-value files but have some differences like sections (ini), details about characters used as separators or to start a comment and the explicit check for unicode escape codes (properties). I think both of them handle their respective (simple) syntaxes fine since I do not know any breaking cases. Also the error handling is on the same level (but could be further improved, as always).

**Comment by utoddl** [comment]
I just dropped pull request #29 in the hopper to provide better error handling in `.ini` files.
